requests/packages/idna/codec.py

```diff
@@ -0,0 +1,118 @@
+from idna.core import encode, decode, alabel, ulabel, IDNAError
```

Comment:
> In this case this should be simple enough I think: `sed "s/from idna/from /"` for each file in the idna directory should do the trick.

That will absolutely work for this sample. If @kjd, however, starts importing modules, e.g.,

```py
from idna import codecs
```

Then that'll result in

```py
from import codecs
```

So it won't quite work and we'll have to be extra careful with this vendor as we update it.